app/controllers/map_controller.rb

```diff
@@ -16,7 +16,11 @@ def index
   end
 
   def show
-    @node = DrupalNode.find_map_by_slug(params[:name]+'/'+params[:date])
+    finder = "#{params[:name]} #{params[:date]}".parameterize
```

Comment:
Umm, well then we can make a `find_map` and `find_note` method for finding notes and maps. Both of them contain different format of friendly_id slug. Overriding the find method may create problems in finding nodes as the find method itself is a friendly_id finder method.